pages/linux/tcpick.md

```diff
@@ -0,0 +1,17 @@
+# tcpick
+
+> It is a command-line tool in Linux used for packet sniffing and network traffic analysis.
+> It allows you to capture and display TCP connections and their data. You can use it to monitor network traffic on a specific interface or capture traffic to and from a particular host or port.
```

Comment:
For shortening the sentence. Feel free to discuss if I delete something important.
```suggestion
> A Linux command-line tool for packet sniffing and network traffic analysis.
> It can capture and display TCP connections and data. It can also monitor network traffic on a specific interface, host, or port.
```

Reply:
@blueskyson Done